src/plugins/ui_stateful/selection.ts

```diff
@@ -546,7 +546,7 @@ export class GridSelectionPlugin extends UIPlugin {
 
     const sheetId = this.getActiveSheetId();
     const handler = new CellClipboardHandler(this.getters, this.dispatch);
-    const data = handler.copy(getClipboardDataPositions(sheetId, target));
+    const data = handler.copy(getClipboardDataPositions(sheetId, target), true);
```

Comment:
nitpick: maybe be should change the boolean arg  isCutOperation to `"copy" | "paste"` so random calls to the method are more readable